client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -183,16 +183,24 @@ private void submitRetryPushData(
     int partitionId = loc.getId();
     if (!revive(
         applicationId, shuffleId, mapId, attemptId, partitionId, loc.getEpoch(), loc, cause)) {
-      wrappedCallback.onFailure(
-          new CelebornIOException("Revive Failed, remain revive times " + remainReviveTimes));
+      logger.error(
```

Comment:
RpcResponseCallback.onFailure will log
```
              logger.error(
                  "Push data to {} failed for shuffle {} map {} attempt {} partition {} batch {}.",
                  loc.hostAndPushPort(),
                  shuffleId,
                  mapId,
                  attemptId,
                  partitionId,
                  nextBatchId,
                  e);
```
will this be duplicate?

Reply:
Yea, should remove this